gcsio/src/main/java/com/google/cloud/hadoop/gcsio/testing/InMemoryGoogleCloudStorage.java

```diff
@@ -90,12 +95,9 @@ private boolean validateBucketName(String bucketName) {
       return false;
     }
 
-    if (bucketName.length() > 63) {
-      return false;
-    }
+    return bucketName.length() <= 63;
```

Comment:
Not on your change.. but it's really weird seeing this random number pops up.  maybe move the comment down below before this line?

Reply:
I will do it in follow-up CL, but this is a testing code, so it's not so critical.